code/perception/src/visualize_filter_comparison.py

```
@@ -0,0 +1,282 @@
+#!/usr/bin/env python
+
+import matplotlib.pyplot as plt
+from math import pi
+
+NEW_FILTER_FILE_NAME = "00"
+OLD_FILTER_FILE_NAME = "00"
+GT_FILE_NAME = "00"
+
+
+# open the file with the estimated positions of the new filter
+nf_pos_file = open(
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)